modules/drivers/ili9341/modIli9341.c

```diff
@@ -534,6 +545,22 @@ void ili9341Begin(void *refcon, CommodettoCoordinate x, CommodettoCoordinate y,
 
 	xMin = x + MODDEF_ILI9341_COLUMN_OFFSET;
 	yMin = y + MODDEF_ILI9341_ROW_OFFSET;
+	
+	switch(sd->rotation) {
```

Comment:
Tiny detail -- coding style inconsistency. There should be space between `switch` and `(`.